{"name": "Проверка валидности поля Повтор пароля", "status": "failed", "statusDetails": {"message": "selene.core.exceptions.TimeoutException: Message: \n\nTimed out after 10s, while waiting for:\nbrowser.element(('xpath', \"//div[contains(text(),'недопустимый пароль')]\")).text\n\nReason: NoSuchElementException: Message: no such element: Unable to locate element: {\"method\":\"xpath\",\"selector\":\"//div[contains(text(),'недопустимый пароль')]\"}\n  (Session info: chrome=111.0.5563.65)\n\nScreenshot: file://C:\\Users\\Qasquad1\\.selene\\screenshots\\1678700584667\\1678700584668.png\nPageSource: file://C:\\Users\\Qasquad1\\.selene\\screenshots\\1678700584667\\1678700584668.html", "trace": "..\\venv\\lib\\site-packages\\selene\\core\\wait.py:102: in for_\n    return fn(self._entity)\n..\\venv\\lib\\site-packages\\selene\\core\\wait.py:56: in __call__\n    return self._fn(entity)\n..\\venv\\lib\\site-packages\\selene\\core\\query.py:47: in <lambda>\n    text: Query[Element, str] = Query('text', lambda element: element().text)\n..\\venv\\lib\\site-packages\\selene\\core\\entity.py:179: in __call__\n    return self._locator()\n..\\venv\\lib\\site-packages\\selene\\core\\locator.py:34: in __call__\n    return self._locate()\n..\\venv\\lib\\site-packages\\selene\\core\\entity.py:1529: in <lambda>\n    f'{self}.element({by})', lambda: self.driver.find_element(*by)\n..\\venv\\lib\\site-packages\\selenium\\webdriver\\remote\\webdriver.py:976: in find_element\n    return self.execute(Command.FIND_ELEMENT, {\n..\\venv\\lib\\site-packages\\selenium\\webdriver\\remote\\webdriver.py:321: in execute\n    self.error_handler.check_response(response)\n..\\venv\\lib\\site-packages\\selenium\\webdriver\\remote\\errorhandler.py:242: in check_response\n    raise exception_class(message, screen, stacktrace)\nE   selenium.common.exceptions.NoSuchElementException: Message: no such element: Unable to locate element: {\"method\":\"xpath\",\"selector\":\"//div[contains(text(),'недопустимый пароль')]\"}\nE     (Session info: chrome=111.0.5563.65)\n\nDuring handling of the above exception, another exception occurred:\ntest_login.py:122: in test_field_password\n    page.registration_field_repeat_password()\n..\\pages\\home.py:295: in registration_field_repeat_password\n    message_error = self.get_element_text(self.message_error_repeat_password_text_var, \"\")\n..\\pages\\base.py:16: in get_element_text\n    return element.get(query.text)\n..\\venv\\lib\\site-packages\\selene\\core\\entity.py:116: in get\n    return self.wait.for_(query)\n..\\venv\\lib\\site-packages\\selene\\core\\wait.py:125: in for_\n    raise self._hook_failure(failure)\nE   selene.core.exceptions.TimeoutException: Message: \nE   \nE   Timed out after 10s, while waiting for:\nE   browser.element(('xpath', \"//div[contains(text(),'недопустимый пароль')]\")).text\nE   \nE   Reason: NoSuchElementException: Message: no such element: Unable to locate element: {\"method\":\"xpath\",\"selector\":\"//div[contains(text(),'недопустимый пароль')]\"}\nE     (Session info: chrome=111.0.5563.65)\nE   \nE   Screenshot: file://C:\\Users\\Qasquad1\\.selene\\screenshots\\1678700584667\\1678700584668.png\nE   PageSource: file://C:\\Users\\Qasquad1\\.selene\\screenshots\\1678700584667\\1678700584668.html"}, "steps": [{"name": "Нажать Аккаунт", "status": "passed", "start": 1678700590455, "stop": 1678700590522}, {"name": "Нажать Зарегистрироваться", "status": "passed", "start": 1678700590522, "stop": 1678700590680}, {"name": "Валидация поля Повторить Пароль", "status": "failed", "statusDetails": {"message": "selene.core.exceptions.TimeoutException: Message: \n\nTimed out after 10s, while waiting for:\nbrowser.element(('xpath', \"//div[contains(text(),'недопустимый пароль')]\")).text\n\nReason: NoSuchElementException: Message: no such element: Unable to locate element: {\"method\":\"xpath\",\"selector\":\"//div[contains(text(),'недопустимый пароль')]\"}\n  (Session info: chrome=111.0.5563.65)\n\nScreenshot: file://C:\\Users\\Qasquad1\\.selene\\screenshots\\1678700584667\\1678700584668.png\nPageSource: file://C:\\Users\\Qasquad1\\.selene\\screenshots\\1678700584667\\1678700584668.html\n\n", "trace": "  File \"C:\\Users\\Qasquad1\\PycharmProjects\\lime-qa\\venv\\lib\\site-packages\\allure_commons\\_allure.py\", line 172, in impl\n    return func(*a, **kw)\n  File \"C:\\Users\\Qasquad1\\PycharmProjects\\lime-qa\\pages\\home.py\", line 295, in registration_field_repeat_password\n    message_error = self.get_element_text(self.message_error_repeat_password_text_var, \"\")\n  File \"C:\\Users\\Qasquad1\\PycharmProjects\\lime-qa\\venv\\lib\\site-packages\\allure_commons\\_allure.py\", line 172, in impl\n    return func(*a, **kw)\n  File \"C:\\Users\\Qasquad1\\PycharmProjects\\lime-qa\\pages\\base.py\", line 16, in get_element_text\n    return element.get(query.text)\n  File \"C:\\Users\\Qasquad1\\PycharmProjects\\lime-qa\\venv\\lib\\site-packages\\selene\\core\\entity.py\", line 116, in get\n    return self.wait.for_(query)\n  File \"C:\\Users\\Qasquad1\\PycharmProjects\\lime-qa\\venv\\lib\\site-packages\\selene\\core\\wait.py\", line 125, in for_\n    raise self._hook_failure(failure)\n"}, "steps": [{"name": "Заполнение поля '' текстом ''wer23''", "status": "passed", "parameters": [{"name": "element", "value": "<selene.core.entity.Element object at 0x0000016AB0584790>"}, {"name": "text", "value": "'wer23'"}, {"name": "fieldName", "value": "''"}], "start": 1678700590680, "stop": 1678700590767}, {"name": "Взять текст из элемента ''''", "status": "failed", "statusDetails": {"message": "selene.core.exceptions.TimeoutException: Message: \n\nTimed out after 10s, while waiting for:\nbrowser.element(('xpath', \"//div[contains(text(),'недопустимый пароль')]\")).text\n\nReason: NoSuchElementException: Message: no such element: Unable to locate element: {\"method\":\"xpath\",\"selector\":\"//div[contains(text(),'недопустимый пароль')]\"}\n  (Session info: chrome=111.0.5563.65)\n\nScreenshot: file://C:\\Users\\Qasquad1\\.selene\\screenshots\\1678700584667\\1678700584668.png\nPageSource: file://C:\\Users\\Qasquad1\\.selene\\screenshots\\1678700584667\\1678700584668.html\n\n", "trace": "  File \"C:\\Users\\Qasquad1\\PycharmProjects\\lime-qa\\venv\\lib\\site-packages\\allure_commons\\_allure.py\", line 172, in impl\n    return func(*a, **kw)\n  File \"C:\\Users\\Qasquad1\\PycharmProjects\\lime-qa\\pages\\base.py\", line 16, in get_element_text\n    return element.get(query.text)\n  File \"C:\\Users\\Qasquad1\\PycharmProjects\\lime-qa\\venv\\lib\\site-packages\\selene\\core\\entity.py\", line 116, in get\n    return self.wait.for_(query)\n  File \"C:\\Users\\Qasquad1\\PycharmProjects\\lime-qa\\venv\\lib\\site-packages\\selene\\core\\wait.py\", line 125, in for_\n    raise self._hook_failure(failure)\n"}, "parameters": [{"name": "element", "value": "<selene.core.entity.Element object at 0x0000016AB05682E0>"}, {"name": "allureText", "value": "''"}], "start": 1678700597828, "stop": 1678700608353}], "start": 1678700590680, "stop": 1678700608355}], "attachments": [{"name": "Скринтшот", "source": "8fe2e39a-ba26-4dd5-aa17-39b412dada1c-attachment.png", "type": "image/png"}, {"name": "Логи консоли", "source": "83f48fc7-697e-4c63-8032-49680463270d-attachment.json", "type": "application/json"}], "start": 1678700590455, "stop": 1678700608356, "uuid": "fc8cf7c8-2dac-44e5-bcf1-f0aeada06432", "historyId": "a99d02fda9c30122d69bbe61b55c08a2", "testCaseId": "95150aaeb7528162060866b2ece24cb3", "fullName": "tests.test_login.TestLogin#test_field_password", "labels": [{"name": "feature", "value": "Тесты логина"}, {"name": "tag", "value": "@pytest.mark.usefixtures('setup')"}, {"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_login"}, {"name": "subSuite", "value": "TestLogin"}, {"name": "host", "value": "DESKTOP-VLT8PU4"}, {"name": "thread", "value": "12544-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_login"}]}